docs/getting-started.md

```diff
@@ -0,0 +1,20 @@
+---
+id: getting-started
+title: Getting Started
+sidebar_label: Getting Started
+---
+
+> The docs are being worked on and more content will be added soon!
+
+Below are some resources for getting started with Clay. To get started with a simple Clay instance you can [check out the starter repo](https://github.com/clay/clay-starter). If you see a phrase you don't recognize then check the [Glossary](/docs/glossary).
+
```

Comment:
We got `404` in this links 😅 
You can use the name of the docs on the links
`[Glossary](glossary.md)`